contracts/interfaces/IUniswapV3SwapRouter.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity >=0.5.17;
+pragma experimental ABIEncoderV2;
+
+interface IUniswapV3SwapRouter {
```

Comment:
can we import https://github.com/Uniswap/v3-periphery/blob/main/contracts/interfaces/ISwapRouter.sol instead

Reply:
if it will compile

Reply:
It will compile for this, but this interface is from Dex Selector so I am just re-using it. Dex Selector needed a custom interface as the swap implementation is in 0.5.17 and ISwapRouter.sol does not support it.